src/openzaak/components/zaken/tests/admin/test_zaak_admin.py

```diff
@@ -343,3 +343,19 @@ def test_zaak_geometrie_change_from_admin(self):
 
         self.assertAlmostEqual(zaak.zaakgeometrie.x, long)
         self.assertAlmostEqual(zaak.zaakgeometrie.y, lat)
+
+    def test_zaak_opschorting_indicatie_sets_eerdere_opschorting(self):
+        zaak = ZaakFactory.create()
```

Comment:
Might be good to explicitly initialize these attributes as `False`, as a sanity check
```suggestion
        zaak = ZaakFactory.create(opschorting_indicatie=False, opschorting_eerdere_opschorting=False)
```